vip-application/src/main/java/fr/insalyon/creatis/vip/application/client/view/monitor/SimulationsToolStrip.java

```diff
@@ -400,6 +409,57 @@ public void onSuccess(Void result) {
         modal.show("Marking selected executions completed...", true);
     }
 
+    private void makeExecutionsPublic() {
+        ListGridRecord[] records = getSimulationsTab().getGridSelection();
+        List<Triplet<String, String, String>> workflowsData = new ArrayList<>();
+        List<String> authors = new ArrayList<>();
+        PublicExecution publicExecution;
+
+        if (records.length == 0) {
+            Layout.getInstance().setWarningMessage("You must select at least 1 workflow!");
+        } else {
+            for (ListGridRecord record : records) {
+                SimulationRecord data = (SimulationRecord) record;
+                SimulationStatus status = SimulationStatus.valueOf(data.getStatus());
+
+                if (status != SimulationStatus.Completed) {
+                    Layout.getInstance().setWarningMessage("You must select only completed executions!");
+                    return;
+                } else {
+                    authors.add(data.getUser());
+                    workflowsData.add(new Triplet<String,String,String>(
+                        data.getSimulationId(), data.getApplication(), data.getApplicationVersion()));
+                }
+            }
+            publicExecution = new PublicExecution("new_experience", workflowsData, 
+                PublicExecution.PublicExecutionStatus.REQUESTED, String.join(", ", authors), "", Collections.emptyList(), "");
+
```

Comment:
it's weird to init the publiExecution with dummy/empty stuff here.
I think canMakeExecutionPublic only need the worflow ids list, so only send them, give workflowsData and authors to MakeExecutionPublicTab and let MakeExecutionPublicTab init the rest.